ql/math/matrix.hpp

```diff
@@ -47,6 +46,9 @@ namespace QuantLib {
         Matrix(Size rows, Size columns);
         //! creates the matrix and fills it with <tt>value</tt>
         Matrix(Size rows, Size columns, Real value);
+        //! creates the matrix and fills it with <tt>values</tt>.
+        //! values are used in a <tt>row by row</tt> fashion.
```

Comment:
We could probably be more generic for this constructor do some safety checks, but it does state the order (row by row) it will fill the matrix.